Source/EasyNetQ/QueueConsumerPair.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using EasyNetQ.Consumer;
+using EasyNetQ.Internals;
+using EasyNetQ.Topology;
+
+namespace EasyNetQ
+{
+    public class QueueConsumerPair
+    {
+        #region Factory methods
```

Comment:
I guess it's a matter of taste. I think they add unnecessary clutter, make it harder to read the code as they are collapsed by default. 

Or to quote this SE answer: http://softwareengineering.stackexchange.com/questions/53086/are-regions-an-antipattern-or-code-smell

> **Is there a good use for regions?**
> 
> No. There was a legacy use: generated code. Still, code generation tools just have to use partial classes instead. If C# has regions support, it's mostly because this legacy use, and because now that too many people used regions in their code, it would be impossible to remove them without breaking existent [sic] codebases.
> 
> Think about it as about goto. The fact that the language or the IDE supports a feature doesn't mean that it should be used daily. **StyleCop SA1124rule is clear: you should not use regions. Never.**